### ISSUE TITLE
Staff trying to view RAN099951 (status 13 I think) on web gets white screen 

### ISSUE THREAD

**Issue by LisaMoore1** [bcgov/range-web#324]
HTML for that white screen is https://myrangebc.gov.bc.ca/range-use-plan/291
Haven’t been able to find another that generates this same error or what makes this one different than others.

**Comment by micheal-w-wells** [comment]
![image](https://user-images.githubusercontent.com/955467/69446742-eec55d00-0d09-11ea-8975-ddb8f4acec22.png)
@calebissharp any idea why this only happens for this plan?  bit stumped on this one, at least right now anyway

**Comment by calebissharp** [comment]
@micheal-w-wells Dunno. I'm going to load the plan in the dev version of the frontend though, because the stack traces in a production build are useless.

**Comment by calebissharp** [comment]
Better stack dump:
```
Uncaught TypeError: Cannot read property 'name' of undefined
    at PlantCommunityBox (PlantCommunityBox.js:140)
    at renderWithHooks (react-dom.development.js:15842)
    at mountIndeterminateComponent (react-dom.development.js:18162)
    at beginWork$1 (react-dom.development.js:19380)
    at HTMLUnknownElement.callCallback (react-dom.development.js:363)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:412)
    at invokeGuardedCallback (react-dom.development.js:466)
    at beginWork$$1 (react-dom.development.js:24597)
    at performUnitOfWork (react-dom.development.js:23529)
    at workLoopSync (react-dom.development.js:23507)
    at renderRoot (react-dom.development.js:23182)
    at runRootCallback (react-dom.development.js:22836)
    at react-dom.development.js:11907
    at unstable_runWithPriority (scheduler.development.js:676)
    at runWithPriority$2 (react-dom.development.js:11855)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11902)
    at flushSyncCallbackQueue (react-dom.development.js:11890)
    at scheduleUpdateOnFiber (react-dom.development.js:22694)
    at dispatchAction (react-dom.development.js:16563)
    at _callee$ (index.js:84)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:282)
    at Generator.prototype.<computed> [as next] (runtime.js:116)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
```

Seems to be an issue with the elevation dropdown.

**Comment by LisaMoore1** [comment]
@calebissharp  I wonder if this has anything to do with the iPad requiring that field (mandatory) but web does not.  It used to be mandatory but decision was made to change that.

**Comment by calebissharp** [comment]
@LisaMoore1 Don't think so. I've got a fix in #328

**Comment by LisaMoore1** [comment]
@micheal-w-wells Can you please assign RAN099951 one back to me so that I can test this one? Thx
